core/src/main/java/org/jclouds/concurrent/config/ExecutorServiceModule.java

```diff
@@ -54,6 +58,42 @@
 @ConfiguresExecutorService
 public class ExecutorServiceModule extends AbstractModule {
 
+   private static final Method CREATE_STL;
```

Comment:
Please add a comment explaining that this code works around Guava 18 and 23 compatibility.

Reply:
Done